### PR TITLE
taskkill: add page

### PR DESCRIPTION
**Pull request by pxgamer**
Part of #1579

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/windows/taskkill.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+`taskkill /pid {{process_id}}`
+
+- Terminate a process by its name:
+
+`taskkill /im {{process_name}}`
+
+- Forcefully terminate a specified process:
+
+`taskkill /f`
```
Comment (sbrl) [inline pages/windows/taskkill.md]: How do you specify the process to terminate? Perhaps we could merge this with one of the other examples?

File: pages/windows/taskkill.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- Terminate a process and its child processes:
+
+`taskkill /t`
```
Comment (sbrl) [inline pages/windows/taskkill.md]: Perhaps we could add `/pid {{process_id}}` to this or something for the same reason?

File: pages/windows/taskkill.md (hunk 1 above)
Comment (pxgamer) [reply pages/windows/taskkill.md]: I did think about that, but I wasn't sure as it covers the `/pid` and `/im` earlier. I didn't really want to duplicate stuff.

I guess could it mention that it needs to be combined with a process id or name?

Comment (sbrl) [reply pages/windows/taskkill.md]: Yeah, probably. Unless we combine the two examples, I guess?

Comment (pxgamer) [reply pages/windows/taskkill.md]: Could I add `(must be combined with a process id or name)` to the end of both descriptions?

Comment (agnivade) [reply pages/windows/taskkill.md]: I think it's ok. It is natural to combine the options from previous examples to have more complex examples and is something the users are expected to do. If you say `/f` and `/pid` and `/im` are mentioned earlier, then that means `/f` can be combined with either of them.

File: pages/windows/taskkill.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# taskkill
+
+> Terminate a process by its process id or name.
+
+- Display information about the usage of the command:
```
Comment (agnivade) [inline pages/windows/taskkill.md]: Move this to the end.

TIP: Arrange the examples in the order you expect the users to use them. Usually we keep the `-h` and its variants at the end, as those are expected to let the user know about further usages of the command.

File: pages/windows/taskkill.md (hunk 1 above)
Comment (agnivade) [reply pages/windows/taskkill.md]: Sorry, I think I was not clear. I meant that it was okay as it was before, i.e. without the `(must be combined with a process id or name)`. But this is OK too.

File: pages/windows/taskkill.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+`taskkill /f`
+
+- Terminate a process and its child processes (must be combined with a process id or name):
+
+`taskkill /t`
+
+- Terminate a process on a remote machine:
+
+`taskkill /s {{remote_name}}`
```
Comment (agnivade) [inline pages/windows/taskkill.md]: Shouldn't we need to give the process id or name here too ?

Comment (pxgamer) [reply pages/windows/taskkill.md]: Yes, it can be combined with any of the above commands, so I wasn't sure if it needed to mention it.

Comment (agnivade) [reply pages/windows/taskkill.md]: Dunno, looks confusing to me. Mentioning it thrice is also redundant. Best way is to go with @sbrl's original recommendation. Use `/pid` for `/f`, `/im` for `/t`, and again `/pid` for `/s`. That wil clearly indicate that pid or process_name can be mixed. And remove the text in `()`.

Comment (pxgamer) [reply pages/windows/taskkill.md]: Sounds good to me. :+1: I'll make that change in a bit.

**Pull request merged**